persistence/sql/persister_continuity.go

```diff
@@ -50,3 +51,20 @@ func (p *Persister) DeleteContinuitySession(ctx context.Context, id uuid.UUID) e
 	}
 	return nil
 }
+
+func (p *Persister) DeleteExpiredContinuitySessions(ctx context.Context, expiresAt time.Time, limit int) error {
+	// #nosec G201
+	err := p.GetConnection(ctx).RawQuery(fmt.Sprintf(
+		"DELETE FROM %s WHERE id in (SELECT id FROM (SELECT id FROM %s c WHERE expires_at <= ? and nid = ? ORDER BY expires_at ASC LIMIT %d ) AS s )",
```

Comment:
Why is there two sub-queries?

Reply:
I needed to do something similar that is metioned here: https://github.com/ory/kratos/pull/2406/#discussion_r860916009 . Going with a single subselect didn't work for SQLite (I tried it few other ways), but this way works.